content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+---
+date: 2024-04-01
+title: "SpinKube on KinD: Rancher Desktop super powers"
+linkTitle: SpinKube on KinD
+description: >
+  Run SpinKube on KinD thanks to Rancher Desktop Wasm implementation.
+author: Nuno "the Corsair" Do carmo ([@nunixtech](https://twitter.com/nunixtech))
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+The goal of this guide is show a way to bring [SpinKube](https://www.spinkube.dev/) to [KinD](https://kind.sigs.k8s.io/) without the need of a custom image, like the [SpinKube on k3d](https://www.spinkube.dev/docs/spin-operator/quickstart/) example.
+
+Instead, the Rancher Desktop (RD) Spin plugin will be used alongside KinD cluster configuration.
+
+Finally, this guide will have three major sections covering the following:
+
+- KinD configurations: contains the different configurations, depending on your OS
+- RD on WSL: targets only Windows users
+- RD on LIMA: targets both Linux and MacOS users
+
+## Prerequisites
+
+In order to follow this guide, the following applications need to be installed:
+
+- Rancher Desktop v1.13
+  - This is the first version with the Spin plugin
+  - The following preferences are set:
+    - Preferences > Container Engine
+      - containerd is selected
+      - Web Assembly (wasm) is enabled
+    - Preferences > Kubernetes
+      - Kubernetes is disabled
+- KinD v0.23
+  - This is the first version with the `nerdctl` provider
+  - If not yet available, you might need to build it (see [Bonus 1: build KinD](#bonus-1-build-kind))
+
+Concerning the Kubernetes tooling, Rancher Desktop already covers it.
+
+#### Connecting the dots
+
+The reason KinD v0.23 is needed with the `nerdctl` provider is because the Spin plugin only works on Rancher Desktop when `containerD` runtime is selected, instead of `docker`.
```

Comment:
```suggestion
The reason KinD v0.23 is needed with the `nerdctl` provider is because the Spin plugin only works on Rancher Desktop when `containerd` runtime is selected, instead of `docker`.
```